site/templates/common/awards.jet

```diff
@@ -0,0 +1,31 @@
+{{block awardNominations()}}
+  {{ if isset(.AwardCategories) && (len(.AwardCategories) > 0) }}
+    <div class="meta-awards row mt-2p5 meta-awards-all">
+    {{ range .AwardCategories }}
+      <div class="meta-award-nomination mb-2p5 col-12 col-sm-8 col-lg-6 col-xl-4">
+        <span><i class="meta-award-nomination-icon"></i></span>
+        <span class="font-weight-bold">{{ i18n("awards_nominated_for") }} {{ .DisplayLabel }}</span>
+      </div>
+    {{ end }}
+    </div>
+  {{ end }}
+{{end}}
+
+{{block inCompetition()}}
```

Comment:
I think one block per file is better.  Having several blocks here means that when I write an override for `awardNominations`, I also need to copy overrides for the other two blocks.